pkg/cmd/pr/create/create.go

```diff
@@ -696,6 +704,68 @@ func submitPR(opts CreateOptions, ctx CreateContext, state shared.IssueMetadataS
 		return err
 	}
 
+	if opts.DryRun {
+		out := opts.IO.Out
+		if opts.IO.IsStdoutTTY() {
+			fmt.Fprint(out, "Would have created a Pull Request with:\n")
+			fmt.Fprintf(out, "title:\t%s\n", params["title"])
```

Comment:
Can you write a little bit about why you included these fields but excluded others like `BaseBranch`, `HeadBranch`, `MaintainerCanModify` or `Template`?

Reply:
See https://github.com/cli/cli/pull/8376/commits/40747c1e4ac3473904534dda7a7b5a4a19692631 that includes  the support for the `BaseBranch` and `BaseBranch`fields

I'll add support for `maintainerCanModify` shortly.

For `Template`, I'm not sure; I haven't used all the options, but the existing data structure called `params` defined in https://github.com/cli/cli/blob/fcb1576d26f8fbf1304c6381591ae33c991d8025/pkg/cmd/pr/create/create.go#L697-L704.

Reply:
> I'll add support for maintainerCanModify shortly.

See 80934f4a37107ed60991746a12704d71f0de54ee